code/game/objects/structures/deployable_turret.dm

```diff
@@ -1,6 +1,7 @@
-/////// MANNED TURRET ////////
+/////// DEPLOYABLE TURRET (FORMERLY MANNED TURRET) ////////
+//All of this file is five year old shitcode, and I'm too scared to touch more than I have to
 
-/obj/machinery/manned_turret
+/obj/machinery/deployable_turret
```

Comment:
this entire is a fkn mess of copy pasted code from real gun code, this should really be refactored if you actually want to use this in game to be more composite.

Reply:
(instead of copy pasted)

Reply:
Manned turret code is all 5 years old, and I'm not a competent enough coder to go through and refactor everything right now, maybe later on I'll see though

Reply:
if you're making something unavailable suddenly available, then it should probably not be super snowflakey code that is falling apart at the seams

Reply:
wouldn't exactly call it making it available, as its added to a single admin only role, and the code is held together with enough tape and glue that I can refactor it at a later date, when I can be actually good at refactoring things

Reply:
fair enough